src/utilities/gql.ts

```diff
@@ -1687,12 +1687,78 @@ const gql = {
   GET_PLANS_AND_MODELS: `#graphql
     query GetPlansAndModels {
       models: ${Queries.MISSION_MODELS}(order_by: { id: desc }) {
-        id
+        constraint_specification {
```

Comment:
Are `constraint_specification`s necessary in this query?

Reply:
I think the issue was that we were populating the models store from using the results of `getPlansAndModels` on the `plans` page but that query didn't include all of the right info that the models page actually needed. That meant that if you loaded the plans page and then went to the models page you'd see some incomplete info in the table and then eventually the models store subscription would kick in and update the data in the table _at some point_.

Reply:
I see. My concern is that the query doesn't match the `ModelSlim` type that it's returning. Should we just change `getPlansAndModels` to return the full `Model` type?

Reply:
Whoops, used the response from SUB_MODELS instead of SUB_MODEL which was the one actually aligned with ModelSlim. Updated to match ModelSlim.